src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -246,6 +246,8 @@ public function getTestsForParse()
             array('{\'foo\': \'bar\', "bar": \'foo: bar\'}', array('foo' => 'bar', 'bar' => 'foo: bar')),
             array('{\'foo\'\'\': \'bar\', "bar\"": \'foo: bar\'}', array('foo\'' => 'bar', 'bar"' => 'foo: bar')),
             array('{\'foo: \': \'bar\', "bar: ": \'foo: bar\'}', array('foo: ' => 'bar', 'bar: ' => 'foo: bar')),
+            array('{foo:bar: "baz"}', array('foo:bar' => 'baz')),
```

Comment:
@stof This was parsed differently before (it would have been parsed as `['foo' => 'bar: baz']`).

Actually, I am really mixed about what to do here. But I could live with merging it into `master` to be completely safe.